### ISSUE TITLE
lighthouse: fix tests

### ISSUE THREAD

**Issue by selfuryon** [nix-community/ethereum.nix#227]
Hi!
I disabled web3signer downloads in lighthouse and remove current patch (#224) because:
* we disabled tests anyway
* in last release lighthouse merged [PR #4163](https://github.com/sigp/lighthouse/pull/4163/files) in which they started to use [dynamic temp dirs](https://github.com/sigp/lighthouse/pull/4163/files#diff-aaab27eb86a3d85b42ed880faa1a29a8f172b26fb2e256915037144a999e8ab1R61), so they reworked how it works now.  

I think we need to make PR to them with capability to check env var and use that path (if it exists) to web3signer binary without downloading.

**Comment by aldoborrero** [comment]
@selfuryon sounds good to me, will you propose such PR? Let me know

**Comment by selfuryon** [comment]
@aldoborrero yeah, I will do that in several days, it looks like an easy task for the first view
